src/Order/Items/Item.php

```diff
@@ -13,24 +14,26 @@ public function __construct(
         private string $name,
         private string $description,
         private int $quantity,
+        private Price $itemPrice,
         private ?string $sku = null,
         private ?string $imageUrl = null,
-        private ?int $weight = null,
-        private ?WeightUnit $weightUnit = null,
+        private ?Weight $itemWeight = null,
+        private ?array $features = null,
```

Comment:
We can introduce a Feature object that can hold 3 parameters:
- key
- value (string|float|int|bool)
- annotation (optional)

and also we can use the spread operator `...` to make it an array.